app/src/main/java/org/sopt/and/home/components/HomeTopBar.kt

```diff
@@ -63,15 +64,18 @@ fun HomeTopBar(
             )
         }
 
-        Row(
+        LazyRow(
             modifier = Modifier
                 .fillMaxWidth()
                 .padding(vertical = 10.dp),
             horizontalArrangement = Arrangement.spacedBy(14.dp)
         ) {
-            genres.forEach { genre ->
+            items(
+                count = genres.size,
+                key = { genres[it] }
+            ) { index ->
```

Comment:
items 확장함수 중에 List<T> 타입을 인자로 넣으면 T 타입을 그대로 뱉어주는 함수가 있어요. 아래처럼 해도 같은 동작할 수 있습니다. (import 주의)
```suggestion
            items(genres) { genre ->
```

Reply:
세미나에서 이렇게 배웠었는데 안되었던 이유가 import를 잘못해서였군요 ㅠ